precode.sh

```diff
@@ -6,38 +6,57 @@
 #   dir2
 #   dir3
 #       dir4
+mkdir -p ./task/{dir1,dir2}
+mkdir -p ./task/dir3/dir4
```

Comment:
`./` нужна только при запуске скрипта, а в обычных командах можно и без нее
```suggestion
mkdir -p task/dir3/dir4
```

Reply:
Во всем скрипте `./` нужна только в 46 строке